provisioning_templates/snippet/create_users.erb

```diff
@@ -0,0 +1,25 @@
+<%#
+kind: snippet
+name: create_users
+-%>
+# Create all users
+<% [@host.owner].flatten.each do |user| -%>
+<% username = user.login -%>
+useradd <%= username %>
+<% if user.ssh_authorized_keys && user.ssh_authorized_keys.any? -%>
+mkdir <%= username %>/.ssh
```

Comment:
I see you're doing it at the end, as long as this is run under root with reasonable umask that's fine, if we want to reuse that later from e.g. REX we should revisit this since this leaves a windows where someone could inject own authorized key